cas-server-webapp/src/main/webapp/WEB-INF/view/jsp/default/ui/casGenericSuccessView.jsp

```diff
@@ -18,18 +18,25 @@
     under the License.
 
 --%>
-<jsp:directive.include file="includes/top.jsp" />
-  <div id="msg" class="success">
-    <h2><spring:message code="screen.success.header" /></h2>
+
+<%-- CAS Page for Logged In with no Service --%>
+
+<jsp:directive.include file="includes/top.jsp"/>
+
+<div id="msg" class="success">
+    <h2><spring:message code="screen.success.header"/></h2>
+    <br/>
     <c:set var="displayName" value="${principal.attributes.username}"/>
     <c:if test="${empty displayName}">
-      <c:set var="displayName" value="${principal.id}"/>
+        <c:set var="displayName" value="${principal.id}"/>
     </c:if>
     <p><spring:message code="screen.success.success" arguments="${displayName}"/></p>
-    <p><spring:message code="screen.success.security" /></p>
-  </div>
-
-<c:set var="alternativeBottomLogout" value="true" />
+    <p><spring:message code="screen.success.security"/></p>
+</div>
 
-<jsp:directive.include file="includes/bottom.jsp" />
+<c:set var="linkSignIn" value="false"/>
+<c:set var="linkSignOut" value="true"/>
```

Comment:
For generic logged in page (when there is no service), show "Sign Out" and "Back to OSF".